x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/deploy_step/deploy_step.tsx

```diff
@@ -26,74 +26,70 @@ import * as i18n from './translations';
 interface DeployStepProps {
   integrationSettings: State['integrationSettings'];
   result: State['result'];
-  connectorId: State['connectorId'];
 }
 
-export const DeployStep = React.memo<DeployStepProps>(
-  ({ integrationSettings, result, connectorId }) => {
-    const { isLoading, error, integrationFile, integrationName } = useDeployIntegration({
-      integrationSettings,
-      result,
-      connectorId,
-    });
+export const DeployStep = React.memo<DeployStepProps>(({ integrationSettings, result }) => {
```

Comment:
The only change in this file is that I removed the `connectorId` prop which was not being used. The rest is Prettier changing the indentation.